go/libkb/output.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package libkb
+
+import (
+	"io"
+
+	"github.com/keybase/client/go/logger"
+)
+
+func OutputWriter() io.Writer {
```

Comment:
It is called from `client/ui.go` but I can switch that to use logger direct. Tried to preserve the API here, but will remove this and fix the callers.